src/features/presence/__tests__/PresenceAvatarList.test.tsx

```diff
@@ -0,0 +1,149 @@
+                const onAvatarMouseEnter = jest.fn();
+                const wrapper = getWrapper({
+                    onAvatarMouseEnter,
+                });
+
+                expect(
+                    wrapper
+                        .find(Tooltip)
+                        .first()
+                        .prop('isShown'),
+                ).toBe(false);
+
+                // Trigger event on the first avatar
+                wrapper
+                    .find(PresenceAvatar)
+                    .first()
+                    .simulate(event);
+
+                expect(onAvatarMouseEnter).toHaveBeenCalled();
+                expect(
+                    wrapper
+                        .find(Tooltip)
+                        .first()
+                        .prop('isShown'),
+                ).toBe(true);
+            },
+        );
+
+        test.each(['blur', 'mouseleave'])('should hide tooltip when correponding avatar encounters %s event', event => {
+            const onAvatarMouseLeave = jest.fn();
+            const wrapper = getWrapper({
+                onAvatarMouseLeave,
+            });
+
+            // Cause the tooltip to show
+            wrapper
+                .find(PresenceAvatar)
+                .first()
+                .simulate('focus');
+
+            expect(
+                wrapper
+                    .find(Tooltip)
+                    .first()
+                    .prop('isShown'),
+            ).toBe(true);
+
+            // Trigger event on the first avatar
+            wrapper
+                .find(PresenceAvatar)
+                .first()
+                .simulate(event);
+
+            expect(onAvatarMouseLeave).toHaveBeenCalled();
+            expect(
+                wrapper
+                    .find(Tooltip)
+                    .first()
+                    .prop('isShown'),
+            ).toBe(false);
+        });
+
+        test('should pass through additional attributes when specified', () => {
+            const avatarAttr = { 'data-resin-target': 'avatar' };
+            const wrapper = getWrapper({
+                avatarAttributes: avatarAttr,
+                'data-resin-feature': 'presence',
+            });
+
+            expect(wrapper.find('.bdl-PresenceAvatarList').prop('data-resin-feature')).toEqual('presence');
+            expect(
+                wrapper
+                    .find(PresenceAvatar)
+                    .first()
+                    .prop('data-resin-target'),
+            ).toEqual('avatar');
+        });
+
+        test('should correctly render collaborators with additional count when number of collaborators is greater than maxAddionalCollaboratorsNum + maxDisplayedAvatars', () => {
+            const maxDisplayedAvatars = 2;
+            const maxAdditionalCollaborators = 1;
+            const wrapper = getWrapper({ maxAdditionalCollaborators, maxDisplayedAvatars });
+
+            expect(wrapper.exists('.bdl-PresenceAvatarList')).toBe(true);
```

Comment:
Are the `exists` assertions necessary if we test the count/text afterward? If so, can we use something besides their CSS classes?